### PR TITLE
Move serialize to utils module (fixes #105)

### LINKED ISSUE #105: Move serialize from api.js into utils
https://github.com/mozilla/fxpay/blob/d060b537a505490cde40721ebf78b98612499089/lib/fxpay/api.js#L144 seems like a good candidate for moving into utils.

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: lib/fxpay/utils.js [hunk 1]
```diff
@@ -197,4 +197,20 @@ define([
         versionDeprecated);
   };
 
+ /**
+  * Take an object of key value pairs and serialize it
```
Comment (kumar303) [inline lib/fxpay/utils.js]: maybe elaborate that 'serialize' means to turn it into a URL-encoded query string.

**Comment by kumar303** [comment]
I know this didn't have direct tests before but now that it's in utils it would be good to add direct tests. It would be good to cover the URL-encoding both for components (left side of the equal sign) and values.

**Comment by kumar303** [comment]
r+wc

**Comment by muffinresearch** [comment]
> I know this didn't have direct tests before but now that it's in utils it would be good to add direct tests. It would be good to cover the URL-encoding both for components (left side of the equal sign) and values.

Tests were/are already part of the PR.

**Pull request merged**